.husky/post-commit

```diff
@@ -12,5 +12,11 @@ notify() {
 # ensure failed lint exit code passes through sed
 set -o pipefail
 
+branch=$(git branch --show-current)
+# Do NOT run this when rebasing or we can't get the branch
+if [ -z "$branch" ]; then
+  exit 0
+fi
```

Comment:
Great, I was running into exactly this issue and neither `--no-verify` nor `HUSKY_SKIP_HOOKS` worked.